deploy/iso/minikube-iso/package/containerd-bin/config.toml

```diff
@@ -5,6 +5,7 @@ plugin_dir = ""
 disabled_plugins = []
 required_plugins = []
 oom_score = 0
+imports = ["/etc/containerd/config.minikube.toml"]
```

Comment:
do u mind verifying manually if the ISO is building ? on the Bulid I see the ISO is not building

Reply:
I removed that part as I figured out that minikube doesn't use the template from the iso build but instead creates the config.toml as part of the minikube vm start.